operators/t8c-certified/42.54.0-beta.1/manifests/t8c-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,222 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1","kind":"Xl","metadata":{"name":"xl-release"},"spec":{"global":{"externalDbIP": "10.0.2.15","customImageNames": false,"repository": "icr.io/cpopen/turbonomic","tag":"8.12.1"}}}]'
+    capabilities: Basic Install
+    categories: Monitoring
+    certified: "false"
+    containerImage: icr.io/cpopen/t8c-operator@sha256:ec39eb9e3bf1e33099ece324e89f6198015e22498ba1c252ee18ed412e8313c1
+    createdAt: "2023-03-24T00:00:00.000Z"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    description: Turbonomic Workload Automation for Multicloud simultaneously optimizes performance, compliance, and cost in real-time. Workloads are precisely resourced, automatically, to perform while satisfying business constraints.
+    repository: https://github.com/turbonomic/t8c-install/tree/master/operator
+    support: Turbonomic, Inc.
+    olm.skipRange: '>=42.53.0-beta.1 <42.54.0-beta.1'
+  name: t8c-operator.v42.54.0-beta.1
```

Comment:
The name here seems incorrect, I believe it should be `t8c-certified`.

Reply:
But it has never been named that way.   From Day 1 this is the naming convention.

Reply:
I have added the appropriate label to the pr.